configs/swarm/docker-compose.yml

```diff
@@ -1,46 +1,60 @@
-# This file is generated by iroha_swarm.
-# Do not edit it manually.
+# target/release/iroha_swarm -p 4 -s xyzzy -c ./configs/swarm -i hyperledger/iroha:dev -o ./configs/swarm/docker-compose.yml -HF
```

Comment:
There was a discussion about putting all parameters into the banner: https://github.com/hyperledger/iroha/pull/3872#discussion_r1318327619

I am not sure if it is a good idea still, as it depends on CWD and contains `target/release/iroha_swarm`, which is environment-dependent.  Option `-F` (`--force`) also doesn't make sense here.

Reply:
What if the cwd is included relative to the swarm binary?

```
# ../../..
# target/release/iroha_swarm ...
#
# [...] from the above working directory and settings
```

Reply:
I'll probably change this to only include the seed as the first line